network/src/peer_store/browser.rs

```diff
@@ -67,15 +69,14 @@ pub struct Storage {
 impl Storage {
     pub async fn new<P: AsRef<Path>>(path: P) -> Self {
```

Comment:
The path needs to be added with the network name. No matter what network is passed in, the same path is used.

Reply:
- For the first question, the user can customize store path in network configuration to make different network use different databases
- For the second, IndexedDB limits store creation only to the time of upgrading database, if we want to use different stores for different configurations, we'll need a suitable way to determine database version